script/utils/analyzeCoverage.ts

```diff
@@ -0,0 +1,321 @@
+/**
+ * Contract Coverage Analysis Script
+ *
+ * This script analyzes test coverage from lcov.info file and categorizes contracts
+ * by their line coverage percentage. It's designed to help track OKR progress
+ * for contract test coverage goals.
+ *
+ * Features:
+ * - Parses lcov.info file to extract coverage data for each contract
+ * - Categorizes contracts into above/below threshold coverage
+ * - Calculates percentage of included contracts above threshold
+ * - Supports excluding specific contract folders from analysis
+ * - Provides detailed line-by-line coverage statistics
+ *
+ * Usage:
+ *   bun run script/utils/analyzeCoverage.ts [--exclude folder1,folder2] [--threshold 90]
+ *
+ * Examples:
+ *   bun run script/utils/analyzeCoverage.ts
+ *   bun run script/utils/analyzeCoverage.ts --threshold 95
+ *   bun run script/utils/analyzeCoverage.ts --exclude "Libraries/LibBytes,Helpers/ExcessivelySafeCall"
+ *   bun run script/utils/analyzeCoverage.ts --threshold 95 --exclude "Libraries/LibBytes"
+ *
+ * Package.json shortcut:
+ *   bun okr:contract-coverage-above-90
+ *
+ * Prerequisites:
+ * - Run "forge coverage" first to generate lcov.info file
```

Comment:
Could we enhance the script to run the `forge coverage` command itself maybe via `bun coverage` command? This would streamlie the process a bit but I am also good to leave it as is